packages/quantic/force-app/main/default/lwc/quanticFacetManager/quanticFacetManager.js

```diff
@@ -0,0 +1,153 @@
+import {
+  initializeWithHeadless,
+  registerComponentForInit,
+} from 'c/quanticHeadlessLoader';
+import {api, LightningElement} from 'lwc';
+
+/** @typedef {import("coveo").FacetManager} FacetManager */
+/** @typedef {import("coveo").SearchStatus} SearchStatus */
+/** @typedef {import("coveo").SearchEngine} SearchEngine */
+
+/**
+ * The `QuanticFacetManager` component acts as a container component allowing facets to be reordered dynamically as search queries are performed.
+ *
+ * An item template element can be assigned to the `itemTemplate` slot allowing to customize the element that wraps each facet.
```

Comment:
iirc @jpmarceau has told me that double newline is better for him when treating things as separate paragraphs.